### PR TITLE
Add ET prod to flux

### PR DESCRIPTION
**Pull request by jthmcts**
### Change description ###
Add ET prod to flux

CR: CHG5008762

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[x] No
```

### CODE REVIEW

**Review by gauravhmcts — APPROVED** [review]

**Review by petermoores-version1 — DISMISSED** [review]

File: k8s/namespaces/et/et-cos/prod.yaml [hunk 1]
```diff
@@ -0,0 +1,18 @@
+apiVersion: helm.fluxcd.io/v1
```
Comment (adusumillipraveen) [inline k8s/namespaces/et/et-cos/prod.yaml:1]: You shouldn't be needing anything i thought on k8s directory  as everything seems to be  flux v2 ?

Comment (petermoores-version1) [reply k8s/namespaces/et/et-cos/prod.yaml:1]: @adusumillipraveen - The repo ReadMe states "Prod | Flux V1 & V2 | Migration in progress". We therefore assumed both V1 & V2 changes are included. We can remove V1 code from the PR if you can update the ReadMe?

Comment (adusumillipraveen) [reply k8s/namespaces/et/et-cos/prod.yaml:1]: I think the status is to reflect that some apps still use v1 as migration is in progress, but new teams always go to v2. One app cannot be maintained on both. @hannah38 pls correct me if wrong.

Comment (hannah38) [reply k8s/namespaces/et/et-cos/prod.yaml:1]: Hi yes, that's right. It says migration in progress to indicate that some teams are still to be migrated - but new teams should be flux v2 only.

Comment (petermoores-version1) [reply k8s/namespaces/et/et-cos/prod.yaml:1]: @adusumillipraveen @hannah38 - Okay no problem I'll remove the Flux V1 code from the PR and let you know once done.

Comment (petermoores-version1) [reply k8s/namespaces/et/et-cos/prod.yaml]: @adusumillipraveen - Okay Praveen, now removed Flux V1 shell script changes from the PR. Please can you review again, thanks.

Comment (thomast1906) [reply k8s/namespaces/et/et-cos/prod.yaml]: @petermoores-version1 - in another PR , could you cleanup/move the other envs to flux v2 as well?

**Review by adusumillipraveen — APPROVED** [review]

**Review by petermoores-version1 — APPROVED** [review]

**Pull request merged**